server/build_apps_test.go

```diff
@@ -9,55 +9,55 @@ import (
 func TestGetExpectedArtifacts(t *testing.T) {
 	job := BuildAppJob{
 		JobName:           "a",
-		RepoName:          "honk",
+		RepoName:          "repo",
```

Comment:
dont need to update those things

Reply:
The test becomes hard to read for me. I already fell over it twice and was wondering what "honk" meant. Shouldn't the test be easy to read?

Reply:
ok